concourse/tasks/run-task.sh

```diff
@@ -88,8 +88,12 @@ aws ecs wait tasks-stopped --tasks $task_id --cluster $CLUSTER
 echo "task finished."
 task_results=$(aws ecs describe-tasks --tasks $task_id --cluster $CLUSTER)
 
-container_id=$(echo $task_results | jq '.tasks[0].containers[]  | select(.name=="app") | .runtimeId')
-echo "Check Splunk for logs: https://gds.splunkcloud.com/en-GB/app/gds-006-govuk/search?q=search%20index%3D%22govuk_replatforming%22%20container_id%3D$container_id"
+if [[ "${LOG_TO_SPLUNK:-true}" == "true" ]]; then
```

Comment:
Could we override the log config in `containerOverrides` so that all tasks run with this `run-task` script use the `awslogs` log driver?

I think the code would be simpler and one wouldn't need to make additional changes (such as setting the `LOG_TO_SPLUNK` flag) to get logs to output to Concourse.

Reply:
I think after our recent conversation that we've determined that it's not possible to override the `logConfiguration` in container overrides - indeed [this is an issue on the ECS containers roadmap](https://github.com/aws/containers-roadmap/issues/74).